internal/security/setup/option/generate.go

```diff
@@ -52,7 +52,7 @@ func Generate() func(*PkiInitOption) (exitCode, error) {
 			return exitWithError, err
 		}
 
-		os.RemoveAll(generatedDirPath)
+		defer os.RemoveAll(generatedDirPath)
```

Comment:
move this `defer` right after the declaration and assignment of `generatedDirPath`